Integrations/src/main/java/io/deephaven/integrations/numpy/Java2NumpyCopy.java

```diff
@@ -230,23 +230,23 @@ public static void copySlice(final Table t, final long rowStart, final long[] da
         copySlice(t, rowStart, nRow, nCol, long.class, "(long)", (cs, k, idx) -> data[idx] = cs.getLong(k));
     }
 
-    //this is commented out so the byte[] method will be called. Both are 8 bit data structures, the boolean cast will
-    //throw an error
-//    /**
-//     * Casts data to the desired type and copies a slice of rows into a flattened 2D array.
-//     * This is useful for copying table data directly into numpy arrays.
-//     *
-//     * @param t        table to copy data from
-//     * @param rowStart first row of data to copy
-//     * @param data     array to copy data into
-//     * @param nRow     number of rows to copy; also the number of rows in <code>data</code>.
-//     * @param nCol     number of table columns; also the number of columns in <code>data</code>.
-//     */
-//    public static void copySlice(final Table t, final long rowStart, final boolean[] data, final int nRow, final int nCol) {
-//        assertCopySliceArgs(t, rowStart, data == null ? -1 : data.length, nRow, nCol);
-//        assert data != null;
-//        copySlice(t, rowStart, nRow, nCol, boolean.class, "(boolean)", (cs, k, idx) -> data[idx] = cs.getBoolean(k));
-//    }
+    /**
+     * Casts data to the desired type and copies a slice of rows into a flattened 2D array.
+     * This is useful for copying table data directly into numpy arrays.
+     *
+     * @param t        table to copy data from
+     * @param rowStart first row of data to copy
+     * @param data     array to copy data into
+     * @param nRow     number of rows to copy; also the number of rows in <code>data</code>.
+     * @param nCol     number of table columns; also the number of columns in <code>data</code>.
+     */
+    public static void copySlice(final Table t, final long rowStart, final boolean[] data, final int nRow, final int nCol) {
+        assertCopySliceArgs(t, rowStart, data == null ? -1 : data.length, nRow, nCol);
+        assert data != null;
+        copySlice(t, rowStart, nRow, nCol, boolean.class, "(boolean)", (cs, k, idx) -> data[idx] = cs.getBoolean(k));
+    }
```

Comment:
This comment seems to indicate that these constructors were intentionally removed so that byte[] is used.  Does the boolean cast throw an error?  Are you certain this code is executed?

Reply:
Looking at the code, I think there is a test.  Possibly the concern is `Java2NumpyCopy.java:382`.  It looks like we automatically cast from type A to type B.  Maybe there is something funny there.  It isn't obvious to me what the problem is.  Just make certain that the unit tests are getting run.  You can also run them with code coverage in IJ to make sure that the code is getting executed.